uaa/src/main/webapp/WEB-INF/spring/saml-providers.xml

```diff
@@ -41,6 +41,14 @@
             </security:filter-chain-map>
         </bean>
 
+        <bean id="idmClient" class="com.vmware.identity.idm.client.CasIdmClient">
```

Comment:
we already have idm client somewhere, can we keep that single copy ? or dis this just move from there?